src/Schema/SchemaException.php

```diff
@@ -145,6 +146,20 @@ public static function sequenceDoesNotExist($sequenceName)
         return new self("There exists no sequence with the name '" . $sequenceName . "'.", self::SEQUENCE_DOENST_EXIST);
     }
 
+    /**
+     * @param string $constraintName
+     * @param string $table
+     *
+     * @return \Doctrine\DBAL\Schema\SchemaException
+     */
+    public static function uniqueConstraintDoesNotExist($constraintName, $table)
+    {
+        return new self(
+            sprintf("There exists no unique constraint with the name '%s' on table '%s'.", $constraintName, $table),
```

Comment:
Please use double quotes to quote names (see https://github.com/doctrine/dbal/pull/3536).

Reply:
All other methods here are using single quotes, I'm just following the standard.

Reply:
We need to make sure that the standard set in master is followed as well during merging up.

Reply:
Done